hw_decorator.py

```diff
@@ -13,33 +13,65 @@
 # «Function executed with counter = {}, function result = {}» и количество раз сколько эта функция выполнялась.
 # Если значение берется из переменной cache, вывести сообщение «Used cache with counter = {}» и
 # количество раз обращений в cache.
+cache = {}
+counter = 0
 
 
 def check_module(func_to_decorate):
+    """Check the remnant of division 100 to func. """
     def wrapper(number):
         func_to_decorate(number)
         if 100 % func_to_decorate(number) == 0:
             print('We are OK!')
-        print(f'Bad news guys, we got {100 % func_to_decorate}')
+        else:
+            print(f'Bad news guys, we got {100 % func_to_decorate(number)}')
+        return func_to_decorate(number)
     return wrapper
 
 
 def check_type_integer(func_to_decorate):
+    """Checking the income variable for type and raise the errors. """
     def wrapper(number):
-        try:
-            type(number) is int
-        except number is str:
+        if type(number) is int:
```

Comment:
should use isinstance() instead of type()

https://docs.quantifiedcode.com/python-anti-patterns/readability/do_not_compare_types_use_isinstance.html